protos/sources/vega/vega.proto

```diff
@@ -534,8 +534,10 @@ message Candle {
   string close = 6;
   // Total trading volume during the candle interval.
   uint64 volume = 7;
+  // Total notional value (size * price) traded during the candle interval.
+  uint64 notional = 8;
```

Comment:
We need to add new fields at the end with a new number, otherwise it'll be a breaking change.